docker-compose.yml

```diff
@@ -88,7 +88,12 @@ services:
     volumes:
       # a place for config and plugin code
       - ./flexmeasures-instance/:/usr/var/flexmeasures-instance/:ro
-    command: flexmeasures jobs run-worker --name flexmeasures-worker --queue forecasting\|scheduling
+      - ./flexmeasures-instance/:/app/instance/:ro
+    entrypoint: ["/bin/sh", "-c"]
+    command: 
+    - |
+      pip install -r instance/requirements.txt
```

Comment:
As I wrote in the flexmeasures-entsoe issue, I would prefer that we do not add a second volume (which the other container only gets as gunicorn uses it, see the comments), but make use of the one we already have.

I hope that this should also work:

`pip install -r /usr/var/flexmeasures-instance/requirements.txt`

Reply:
I can give that a try. To me, it would make sense to do the same for the server container to keep things consistent. Do you agree?

Reply:
I can see the consistency argument, but for me it seems clear (also from the comments in the server setup part) that we only need the second to serve web requests (gunicorn). So no need to have it in the worker. For me, efficiency trumps consistency if clarity is there :)

I am traveling, so it might be a while until I test this locally. Probably the simplest test script for this can be quite straight-forward:

- echo a dependency into instance/requirements.txt (is that the right path?)
- docker compose build .
- send a command to the worker container via docker exec which greps the output of pip list for the dependency